spec/models/conference_spec.rb

```diff
@@ -23,6 +28,26 @@
     end
   end
 
+  describe 'chronology' do
+    subject { FactoryGirl.build(:conference) }
+
+    context 'when the dates are in the right order' do
+      it 'is valid' do
+        subject.starts_on = '2016-11-20'
+        subject.ends_on = '2016-11-21'
+        expect(subject).to be_valid
+      end
+    end
+
+    context 'when its end date is before start date' do
+      it 'is invalid' do
+        subject.starts_on = '2016-07-15'
+        subject.ends_on = '2016-07-07'
+        expect(subject).not_to be_valid
```

Comment:
I think it's `shoulda-matchers` that provides a better check: 
```ruby
expect(subject).to 

- have(1).error_on(:ends_on)

```
(or some such, I haven't checked the API). It's better because the model can be invalid for other reasons than the start/end date order.